Makefile

```diff
@@ -231,29 +231,42 @@ presubmit:   ## Regenerate all resources, build all images and run all tests.
 presubmit: updateversions regen bin test
 
 .PHONY: updateversions
-CURRENT_TAG = v0.9.0-gke.1
+export CURRENT_TAG = v0.9.0-gke.1
+export CURRENT_PROM_TAG = v2.41.0-gmp.9-gke.0
+export CURRENT_AM_TAG = v0.25.1-gmp.2-gke.0
+export CURRENT_RE_TAG = v0.9.0-gke.1
+export CURRENT_CONFIG_RELOADER_TAG = v0.9.0-gke.1
 #TODO(macxamin) Sync CURRENT_DATASOURCE_SYNCER_TAG with CURRENT_TAG
-CURRENT_DATASOURCE_SYNCER_TAG = v0.9.1-gke.0
-CURRENT_PROM_TAG = v2.41.0-gmp.9-gke.0
-CURRENT_AM_TAG = v0.25.1-gmp.2-gke.0
-LABEL_API_VERSION = 0.9.0
-FILES_TO_UPDATE = $(shell find manifests cmd/operator/deploy -type f -name "*.yaml")
+export CURRENT_DATASOURCE_SYNCER_TAG = v0.9.1-gke.0
```

Comment:
```suggestion
export CURRENT_DATASOURCE_SYNCER_TAG = v0.10.0-gke.3
```

Reply:
Shouldn't this be in a separate change? I didn't change the version here.

Reply:
Nevermind, rebased on main again and that picked it up.